packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3087,17 +3087,21 @@ Polls a push verification transaction for completion. The transaction result is
 {
   "expiresAt": "2015-04-01T15:57:32.000Z",
   "factorResult": "WAITING",
+  "profile":{
+     "credentialId":"jane.doe@example.com",
+     ...
```

Comment:
Hm...actually, I guess based on the synxtax/error highlighting here, github doesn't seem to like this. Though also it seems like the CI thinks it's fine. I'll leave it up to doc team, but as Sanjay and I discussed out of band, it's our preference to send minimal information (so as not to be over-explicit with the profile details).